01-Contracts/test/SteamExchange.test.js

```diff
@@ -10,423 +10,435 @@ const traveler = require("ganache-time-traveler");
 const TEST_TRAVEL_TIME = 3600 * 2; // 1 hours
 
 describe("StreamExchange", () => {
-    const errorHandler = (err) => {
-        if (err) throw err;
-    };
-
-    const names = ["Admin", "Alice", "Bob"];
-
-    let sf;
-    let dai;
-    let ethx;
-    let usd;
-    let usdcx;
-    let ric;
-    let app;
-    let tp; // Tellor playground
-    let usingTellor;
-    let sr; // Mock Sushi Router
-    let ricAddress = "0x263026e7e53dbfdce5ae55ade22493f828922965";
-    const u = {}; // object with all users
-    const aliases = {};
-    let owner;
-    let alice;
-    let bob;
-    const RIC_TOKEN_ADDRESS = "0x263026E7e53DBFDce5ae55Ade22493f828922965"
-    const SUSHISWAP_ROUTER_ADDRESS = "0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506"
-    const TELLOR_ORACLE_ADDRESS = "0xC79255821DA1edf8E1a8870ED5cED9099bf2eAAA"
-    const TELLOR_REQUEST_ID = 1
-    const BOB_ADDRESS = "0x00Ce20EC71942B41F50fF566287B811bbef46DC8"
-    const ALICE_ADDRESS = "0x9f348cdD00dcD61EE7917695D2157ef6af2d7b9B"
-    const OWNER_ADDRESS = "0x3226C9EaC0379F04Ba2b1E1e1fcD52ac26309aeA"
-    const SF_REG_KEY = process.env.SF_REG_KEY
-    let oraclePrice;
-
-    var appBalances = {ethx: [], daix: [], usdcx: [], ric: []}
-    var ownerBalances = {ethx: [], daix: [], usdcx: [], ric: []}
-    var aliceBalances = {ethx: [], daix: [], usdcx: [], ric: []}
-    var bobBalances = {ethx: [], daix: [], usdcx: [], ric: []}
-
-    before(async function () {
-        //process.env.RESET_SUPERFLUID_FRAMEWORK = 1;
-        let response = await axios.get('https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd')
-        oraclePrice = parseInt(response.data.ethereum.usd * 1.005 * 1000000).toString()
-        console.log("oraclePrice", oraclePrice)
-    });
-
-    beforeEach(async function () {
-      // Admin
-      await hre.network.provider.request({
-        method: "hardhat_impersonateAccount",
-        params: [OWNER_ADDRESS]}
-      )
-      owner = await ethers.provider.getSigner(OWNER_ADDRESS)
-      await deployFramework(errorHandler, {
-          web3,
-          from: owner.address,
-      });
```

Comment:
This should be removed as we are using Polygon mainnet SF framework. This code is useful for deploying SF test framework only.